spec/custom-gutter-component-spec.coffee

```diff
@@ -0,0 +1,132 @@
+CustomGutterComponent = require '../src/custom-gutter-component'
+Gutter = require '../src/gutter'
+
+describe "CustomGutterComponent", ->
+  [customGutterComponent, gutter] = []
+
+  beforeEach ->
+    mockGutterContainer = {}
+    gutter = new Gutter(mockGutterContainer, {name: 'test-gutter'})
+    customGutterComponent = new CustomGutterComponent({gutter})
```

Comment:
Nitpick: I think you should name this variable `gutterComponent` to match up with `gutter`.